lib/rubocop/cop/betterment/render_status.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require_relative 'utils/response_status'
+
+module RuboCop
+  module Cop
+    module Betterment
+      class RenderStatus < Base
+        extend AutoCorrector
+
+        include Utils::ResponseStatus
+
+        MSG = <<~MSG.gsub(/\s+/, " ")
+          Did you forget to specify an HTTP status code? The default is `status: :ok`, which might
+          be inappropriate in this situation. Rendering after a POST, PUT, PATCH or DELETE request
+          typically represents an error (e.g. `status: :unprocessable_entity`).
+        MSG
+
+        def on_def(node)
+          each_offense(node, :render) do |responder|
+            add_offense(responder) do |corrector|
+              corrector.insert_after(responder, ", status: #{infer_status(responder).inspect}")
+            end
+          end
+        end
+
+        private
+
+        def infer_status(responder)
+          case extract_template(responder).to_s
+          when 'new', 'edit'
+            :unprocessable_entity
+          else
+            :ok
+          end
+        end
```

Comment:
This is new, and I think makes autocorrection of `render` a bit safer/more reliable. If you don't specify the status for `render`, we'll check what template you're rendering. If you're rendering the `new` or `edit` template, we'll assume `:unprocessable_entity`. Otherwise, we'll assume `:ok`.